src/backend/columnar/sql/columnar--11.0-1--11.0-2.sql

```diff
@@ -0,0 +1,29 @@
+-- columnar--11.0-1--11.0-2.sql
```

Comment:
as 11.0-1 isn't released yet, I would expect this to be in 11.0-1

Reply:
yeah makes sense